packages/core/src/amazonq/lsp/workspaceInstaller.ts

```diff
@@ -31,7 +32,11 @@ export class WorkspaceLSPResolver implements LspResolver {
         const nodePath = path.join(installationResult.assetDirectory, nodeName)
         await fs.chmod(nodePath, 0o755)
 
-        await cleanLspDownloads(manifest.versions, path.basename(installationResult.assetDirectory))
+        const deletedVersions = await cleanLspDownloads(
+            manifest.versions,
+            path.basename(installationResult.assetDirectory)
+        )
+        logger.debug(`cleaning old LSP versions deleted ${deletedVersions.length} versions`)
```

Comment:
nitpick
```suggestion
        logger.debug(`Cleaned up ${deletedVersions.length} old versions`)
```
what are your thoughts?